### ISSUE TITLE
After update to 2.0.7 pointer is inaccurate

### ISSUE THREAD

**Issue by urioTV** [KaizIqbal/Bibata_Cursor#173]
After the update, the cursor isn't aligning correctly, especially in QT applications. For instance, when trying to click a checkbox, I have to click slightly to the left of the box rather than directly on it. Interestingly, this issue doesn't affect everything—it seems to occur mostly in QT apps but not exclusively.

I'm using KDE 6.2.3

**Comment by tuftman-0** [comment]
I also have this bug
I'm running Hyprland 0.45.2

**Comment by Wizdore** [comment]
Also facing the same issue, only the default pointer icon is off.
Im using Hyprland 0.45.2

**Comment by SqrtMz** [comment]
same problem, from hyprland 0.45.2-3 (From Arch repo), and version 2.0.7 of Bibata (from Unstable Nix Repo)... Solved by downgrading to Nix Stable (Bibata 2.0.6), so yeah, a problem from the package version

**Comment by 0x57e11a** [comment]
also having this issue on hyprland 0.45.0, cursor v2.0.7 from unstable nixpkgs

**Comment by tuftman-0** [comment]
yeah also clarifying I am using bibata v2.0.7 from unstable nixpkgs

**Comment by SqrtMz** [comment]
the problem was apparently Nix only (it's been fixed for a while), ig the issue can be closed
